app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ActivitiesController < ApplicationController
+
+  before_action :set_project
+
+  def story
```

Comment:
Yes, it should only retrieve the activities related to a single Story. I've used `story` because I've seen that there are activities related to a Project. :) Do you have any other naming suggestion?